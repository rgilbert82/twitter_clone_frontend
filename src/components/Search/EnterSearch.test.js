import React from 'react';
import { EnterSearch } from '.';

describe(EnterSearch, () => {
  let component;

  beforeEach(() => {
    component = <EnterSearch />;
  });

  it("renders properly", () => {
    expect(component).toMatchSnapshot();
  });
});
