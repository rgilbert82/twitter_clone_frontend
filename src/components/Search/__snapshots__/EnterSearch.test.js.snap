// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`class EnterSearch extends _react2.default.Component {
  render() {
    return (
      _react2.default.createElement("div", { id: "section_message", __source: { fileName: _jsxFileName, lineNumber: 6 }, __self: this },
        _react2.default.createElement("h2", { __source: { fileName: _jsxFileName, lineNumber: 7 }, __self: this }, "Enter a hashtag to search!")));


  }} renders properly 1`] = `<EnterSearch />`;
